airbyte-integrations/connectors/source-postgres/src/test-integration/java/io/airbyte/integrations/io/airbyte/integration_tests/sources/AbstractPostgresSourceDatatypeTest.java

```diff
@@ -46,15 +46,15 @@ protected void initTests() {
     addDataTypeTestData(
         TestDataHolder.builder()
             .sourceType("bigint")
-            .airbyteType(JsonSchemaType.INTEGER)
+            .airbyteType(JsonSchemaType.INTEGER_V1)
```

Comment:
is this generating a schema like
```json
{
  "airbyte_type": "WellKnownTypes....Integer"
}
```
 ?

(which is probably not what we want)

Reply:
No, the airbyteType is a bit of a misnomer. It actually refers to a `JsonSchemaType`.
https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/bases/standard-source-test/src/main/java/io/airbyte/integrations/standardtest/source/TestDataHolder.java#L199

Can rename this to avoid confusion

Reply:
ah, got it - up to you, I'm not blocking the PR for this but you're the expert on this part of the codebase 🤷